src/main/scala/com/typesafe/sbt/packager/universal/Archives.scala

```diff
@@ -9,6 +9,21 @@ object Archives {
 
   /**
    * Makes a zip file in the given target directory using the given name.
+   * @param target folder to build package in
+   * @param name of output (without extension)
+   * @param mappings included in the output
+   * @param top level directory
+   * @return zip file
+   */
+  @deprecated(
+    "Use [[com.typesafe.sbt.packager.universal.Archives.makeZip(File, String, Seq[(File, String)], Option[String], Seq[String]): File]]",
```

Comment:
@huntc - is there a way to reference overloaded method in Scaladoc? This doesn't seem to come out right.